### ISSUE TITLE
dd(Emun::cases()) consumes too much memory

### ISSUE THREAD

**Issue by kenjis** [kint-php/kint#392]
Kint 4.1.4

> Fatal error: Allowed memory size of 536870912 bytes exhausted (tried to allocate 98101936 bytes) in .../vendor/kint-php/kint/src/Renderer/RichRenderer.php on line 369

```php
dd(GlobalAttributes::cases());

enum GlobalAttributes: string
{
    case accesskey = "Specifies a shortcut key to activate/focus an element";
    case contenteditable = "Specifies whether the content of an element is editable or not";
    case data = "Used to store custom data private to the page or application";
    case dir = "Specifies the text direction for the content in an element";
    case draggable = "Specifies whether an element is draggable or not";
    case hidden = "Specifies that an element is not yet, or is no longer, relevant";
    case id = "Specifies a unique id for an element";
    case lang = "Specifies the language of the element's content";
    case spellcheck = "Specifies whether the element is to have its spelling and grammar checked or not";
    case style = "Specifies an inline CSS style for an element";
    case tabindex = "Specifies the tabbing order of an element";
    case title = "Specifies extra information about an element";
    case translate = "Specifies whether the content of an element should be translated or not";

    public function getType(): bool
    {
        return match ($this) {
            self::contenteditable, self::draggable, self::hidden, self::spellcheck, self::translate => "bool",
            default => "string"
        };
    }
}
```

**Comment by jnvsor** [comment]
Well that was a bit more complicated than I thought it'd be. Fixed in 0518ee9e1b5b3c7bbf3c9ad837a6591e39dbfab0, can you give this a spin to confirm before I push it @kenjis ?

**Comment by kenjis** [comment]
@jnvsor How can I get the commit?

**Comment by jnvsor** [comment]
I'd just download the [phar](https://github.com/kint-php/kint/blob/0518ee9e1b5b3c7bbf3c9ad837a6591e39dbfab0/build/kint.phar) and try it from that

**Comment by kenjis** [comment]
Thanks. It worked fine.
<img width="1175" alt="Screeshot 2022-08-28 6 14 27" src="https://user-images.githubusercontent.com/87955/187048203-abecab96-34a1-40b6-9e51-95480153c378.png">

**Comment by jnvsor** [comment]
Fixed in 0518ee9e1b5b3c7bbf3c9ad837a6591e39dbfab0
